ztag/transforms/ssh.py

```diff
@@ -109,6 +109,12 @@ def _transform_object(self, obj):
         certkey_public_key = grab['key_exchange']['server_host_key']['certkey_public_key'].resolve()
         if certkey_public_key is not None:
             rename_key(certkey_public_key, 'cert_type', 'type')
+            # 2018/09/11: Workaround for mis-typed CertType.id field in ES; actual type is uint32,
+            # current ES type is keyword (string). Added 'exclude={elasticsearch}' to the schema,
+            # but to prevent attempted insertions of mistyped values, we need to actually remove the
+            # id field as well.
+            cert_type = certkey_public_key['type']
```

Comment:
This can throw if `type` isn't present (whereas `rename_key` and `del_key` both handle this silently). We should use a `.get()` and check if not `None`, or find a way to wrap in a try except.